### PR TITLE
Milestone 2: Create interface

### PR DESCRIPTION
**Pull request by NtwaliHeritier**
In this milestone,
- I created the four components
- I created a user interface

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import Display from './display';
+
+function App() {
+  return (
+    <>
+      <Display result="0" />
+      <ButtonPanel />
+    </>
+  );
+}
```
Comment (christianotieno) [inline src/components/App.js]: ## Use ES6 Arrow function

- [ ] kindly implement this function component with the recommended ES6 arrow function.
```suggestion
const  App = () => {
  return (
    <>
      <Display result="0" />
      <ButtonPanel />
    </>
  );
}
```

File: src/components/display.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import PropTypes from 'prop-types';
+
+const Display = props => {
+  const { result } = props;
+  return <h3>{result}</h3>;
+};
```
Comment (christianotieno) [inline src/components/display.js]: ## Optional Change: Destructure this component 
- [ ] This `Display` component should have been destructured as inspired below. You will end up using elegant, easy to read code and with fewer lines of code as well.
```suggestion
const Display = { (result) } => {
  return <h3>{result}</h3>;
};
```

File: src/components/display.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+export default Display;
+
+Display.propTypes = {
+  result: PropTypes.string,
+};
```
Comment (christianotieno) [inline src/components/display.js:15]: ## Add default props in this component
- [ ] This display component should have the incoming prop which is the `result` to be a String with default value = 0 ([should be String screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png)) ([default props screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png))
```suggestion
Display.defaultProps = {
  value: "0"
};

Display.propTypes = {
  result: PropTypes.string,
};
```

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes required ♻️ 
Hello @NtwaliHeritier 👋 
Good job with the milestone so far. 👍 

There aren't many issues in this milestone as you have not deviated from the main objectives much. I am glad you are still in line the requirements and that you are determined to do everything right with this project. ⌛ 

Below 👇 are changes that touch on code destructuring, use of ES6 syntax in declaring functions and issues to do with default props. 

Kindly take a look and wor on the suggestions. Cheers!

### Happy coding 💻 
#### CHristian Otieno (CR)

**Review by BrittanyBlake — APPROVED** [review]
### Status: Approved ✅ 
****
### Hi @NtwaliHeritier ! 👋 
### Great job implementing the requested changes!! 🎖️ It's a 👍 from me! Go ahead and merge this pull request and keep up the good work 😄
****


 Good Luck and Happy Coding! 🍀 👩🏾‍💻

**Pull request merged**